docs/src/pages/components/BottomBar.vue

```diff
@@ -68,6 +68,18 @@
               </md-table-row>
 
               <md-table-row>
+                <md-table-cell>md-src</md-table-cell>
```

Comment:
I think that `md-icon-src` will be more clear. `md-src` could be related to the link src on the bottom bar.